scripts/init_app_dir.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+set -e pipefail
+
+rm -rf "${APP_DIR}"
+
+mkdir -p "${APP_DIR}/usr/bin"
+mkdir -p "${APP_DIR}/usr/lib"
+mkdir -p "${APP_DIR}/usr/qml"
+mkdir -p "${APP_DIR}/usr/i18n"
+mkdir -p "${APP_DIR}/usr/bin/StatusQ"
+mkdir -p "${APP_DIR}/usr/plugins/platforminputcontexts/"
```

Comment:
If all operations are performed in `${APP_DIR}` why can't we just `cd` into it instead of adding `${APP_DIR}` to every single path?

Also, all these can be done with a single `mkdir`.

Reply:
Decided to keep as is, without `cd`.

If I cd and mkdir, need to cd back.
If I continue in that dir will need to ammend copy from paths.

Reply:
Not sure why would you need to `cd` to `mkdir`, but it's not a big deal.
Also, in scripts `pushd`/`popd` is more common for similar reasons.